src/domUpdates.js

```diff
@@ -1,29 +1,292 @@
+      .addEventListener('click', getTripEstimate);
+  }
+};
+
+// ________________NEED TO MOVE TO SCRIPTS and Pull OUT DOM UPDATE INFO TO KEEP IN DOM
+
+export const getTripEstimate = () => {
+  const tripForm = document.getElementById('tripForm');
+  const tripDate = document.getElementById('start').value;
+  const formattedStartDate = dayjs(tripDate).format('YYYY/MM/DD');
+  const duration = document.getElementById('duration').value;
+  const numOfTravelers = document.getElementById('travelers').value;
+
+  if (formattedStartDate === '' || !duration || !numOfTravelers) {
+    cardsGrid.innerHTML += `
+      <div id="errorAdjacentElement">
+        <br>
+        <p class="error" id="formError">PLEASE FILL OUT ALL FORM DATA BEFORE SUBMISSION</p>
+      <div>  
+    `;
+
+    setTimeout(function () {
+      const errorMessage = document.getElementById('formError');
+      errorMessage.remove(0);
+    }, 5000);
+
+    document
+      .getElementById('getEstimate')
+      .addEventListener('click', getTripEstimate);
+    return;
+  }
+
+  if (!formattedStartDate || duration || numOfTravelers) {
+    hide(tripForm);
+
+    tripDetails = {
+      id: tripsData.length + 1,
+      userID: currentTraveler.id,
+      destinationID: destinationSelected.id,
+      travelers: parseInt(numOfTravelers),
+      date: dayjs(tripDate).format('YYYY/MM/DD'),
+      duration: parseInt(duration),
+      status: 'pending',
+      suggestedActivities: [],
+      tripsDestination: destinationSelected,
+    };
+
+    currentTraveler.stagedTrip = new Trip(tripDetails, destinationsData);
+    currentTraveler.stagedTrip.updateTripProperties();
+    const estimate = currentTraveler.stagedTrip.cost;
+
+    cardsGrid.innerHTML += `
+    <div>
+      <p>You're estimated cost is ${estimate} for ${numOfTravelers} travelers for a duration of ${duration} days.</p>
+      <button class= "submit-button" id="confirmTripBtn">Confirm Trip</button>
+    </div>
+
+    `;
+    document
+      .getElementById('confirmTripBtn')
+      .addEventListener('click', postUserTrip);
+  }
+};
+
+const postUserTrip = () => {
+  const tripRequestObject = {
+    id: tripDetails.id,
+    userID: tripDetails.userID,
+    destinationID: tripDetails.destinationID,
+    travelers: tripDetails.travelers,
+    date: tripDetails.date,
+    duration: tripDetails.duration,
+    status: tripDetails.status,
+    suggestedActivities: [],
+  };
+  postData(tripRequestObject, 'trips');
+};
+
+export const displaySuccessfulTripRequest = (response) => {
+  document.getElementById('confirmTripBtn').innerHTML = `
+  <h4>Thank you for having<br>us on your team.</h4>
+  <h3>Your can find your trip<br>in your pending trips</h3>
+  <p>Book another trip!</p>
+  `;
+  // addToPendingTrips(response);
+  // setTimeout(function () {
+  //   renderDestinationsGrid();
+  // }, 6000);
+};
+
+const addToPendingTrips = (approvedTrip) => {
+  let approved = new Trip(approvedTrip);
+  approved.updateTripProperties();
```

Comment:
Maybe take a look at the naming of 'approved' here and change it to approvedTrip? That way you immediately know what that variable is representing in case you need to reuse it somewhere else in the future.